### PR TITLE
Fix syntax for 0.980.0

### PR DESCRIPTION
**Pull request by Anoukh**
## Purpose
This PR will update the syntax according to Ballerina 0.980.0 syntax changes and update README with missing content

### CODE REVIEW

**Comment by pubudu91** [comment]
Shall we fix the typos in the readme as well? e.g., docker -> Docker

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A service composition is an aggregate of services collectively composed to autom
 
 > This guide walks you through the process of implementing a service composition using Ballerina language. 
 
-The following are the sections available in this guide.
+Following are the sections available in this guide.
```
Comment (samgnaniah) [inline README.md]: Please revert this change. Using 'The following' in these instances is correct. Please apply elsewhere when you come across this elsewhere.

See https://dictionary.cambridge.org/grammar/british-grammar/following-or-the-following

File: README.md [hunk 2]
```diff
@@ -140,27 +142,61 @@ service<http:Service> travelAgencyService bind travelAgencyEP {
...
+    outReqPayload.DepartureDate == null || airlinePreference == null ||
+    hotelPreference == null || carPreference == null) {
+    outResponse.statusCode = 400;
+    outResponse.setJsonPayload({"Message":"Bad Request - Invalid Payload"});
+    _ = client -> respond(outResponse);
+    done;
+}
+```
+
+The above code shows how the request JSON payload is parsed to create json literals required for further processing.
```
Comment (samgnaniah) [inline README.md]: Change 'json' to 'JSON'. Apply elsewhere if applicable.

File: README.md [hunk 3]
```diff
@@ -406,11 +437,11 @@ endpoint http:Listener travelAgencyEP {
 service<http:Service> travelAgencyService bind travelAgencyEP {    
 ``` 
 
-- Here we have used ``  @kubernetes:Deployment `` to specify the docker image name which will be created as part of building this service. 
+- Here we have used ``  @kubernetes:Deployment `` to specify the Docker image name which will be created as part of building this service.
```
Comment (samgnaniah) [inline README.md]: Change 'which' to 'that'. Alternately you can add a comma before 'which'. Using 'that' is better considering the context of the sentence.

**Comment by pubudu91** [comment]
Shall we get rid of the complete programming language part in line 40 of readme.md? That doesn't make sense
